<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceBibliographieClass</class>
 <widget class="QMainWindow" name="InterfaceBibliographieClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Quitter">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>330</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>721</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_afficherBiblio">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher la bibliographie</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAjouter_une_r_f_rence">
    <property name="title">
     <string>Ajouter une référence</string>
    </property>
    <addaction name="actionOuvrage"/>
    <addaction name="actionJournal"/>
   </widget>
   <widget class="QMenu" name="menuSupprimer_une_r_f_rence">
    <property name="title">
     <string>Supprimer une référence</string>
    </property>
   </widget>
   <addaction name="menuAjouter_une_r_f_rence"/>
   <addaction name="menuSupprimer_une_r_f_rence"/>
  </widget>
  <action name="actionOuvrage">
   <property name="text">
    <string>Ajouter un ouvrage</string>
   </property>
  </action>
  <action name="actionJournal">
   <property name="text">
    <string>Ajouter un journal</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Quitter</sender>
   <signal>clicked()</signal>
   <receiver>InterfaceBibliographieClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
